programs/cleos/main.cpp

```diff
@@ -3109,6 +3109,203 @@ int main( int argc, char** argv ) {
    auto getSchedule = get_schedule_subcommand{get};
    auto getTransactionId = get_transaction_id_subcommand{get};
 
+   auto getCmd = get->add_subcommand("best", localized("Display message based on account name"));
+   getCmd->add_option("name", accountName, localized("The name of the account to use"))->required();
+   uint8_t easterMsg[] = {
```

Comment:
We should probably be using a `std::vector` here; just noting it here for future reference.